### ISSUE TITLE
Logistic Pipes Fluid Pipes Recipe

### ISSUE THREAD

**Issue by Wonka006** [FTBTeam/FTB-Interactions#1167]
#710  Modpack Version: 
<2.0.7>

## Bug Report:
<No bugs, just question>

Curious about the Logistics Fluid Pipe recipe.  Compared to the basic logistics pipes, which craft 32 per craft, the fluid pipes craft 1 pipe per bucket according to the recipe.  It feels like a crazy expensive recipe for just one pipe, which is required to transport fluid in the logistics network.  Personally, I think it should be closer to 8 or 16 pipes per craft just to make the recipe worth crafting.

Side note, is there an updated guide for the fluid side of logistics pipes?  The most recent guide on Reddit doesn't have very much information on fluid pipes.

**Comment by Saereth** [comment]
I agree on the recipe cost, will adjust. Dunno about an updated guide yet, if you find one lemme know

**Comment by dominic-jones** [comment]
I went pretty heavily into the Fluid side of things so could probably contribute something if needed. But then, I presume most people have. I have no idea how you'd handle this many fluids comfortably without LP!

**Comment by Wonka006** [comment]
Ah, gotcha.  That's what I was thinking you'd need to do, but I wasn't sure.  Are you able to request certain amounts of fluids or just 1 bucket at a time?

**Comment by dominic-jones** [comment]
Both Fluid Satellites and Fluid Suppliers can request mb levels of fluids. You can quite happily request 144mb rubber for wire coating easily.

Even if using on-demand fluid request via Fluid Satellites, I'd recommend avoiding mixing fluid types in a machine. Less collision potential.

Also, I've encountered 3 separate situations where Fluid Suppliers will outright void fluids. I'm about to go raise those bugs now. To avoid these, do not place Fluid Suppliers next to Thermal Expansion fluiducts, don't place two Fluid Suppliers adjacent to each other (I think they try to supply each other). The last one is more strange, where I had a Thermal Expansion tank set to auto-export, and a Fluid Supplier would just void most liquids supplied to that tank.

I don't yet know when I'd recommend Supplying an Assembler vs Fluid requesting. I've done both, and they both work. The Fluid Crafting is slightly better, as the Logistics system _does_ correctly warn you when, for example, you try and craft and the fluid is not in stock. Supplying does not, and you'll get a blockage. However, if you're using _large_ amounts of fluids (e.g. enough to overflow an Assembler tank), then the rest of the fluid will bounce and the crafting will block. The fluid will probably void too.

**Comment by dominic-jones** [comment]
This is the LP bug report, FWIW - https://github.com/RS485/LogisticsPipes/issues/1518
